vagrant/Vagrantfile

```diff
@@ -72,13 +72,30 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             end		
           end		
         end
+        if ENV['PYTHON_REPOS']
+          config.vm.provision "file", source: "./rackhd-pm2-config.yml", destination: "~/rackhd-pm2-config.yml"
+          config.vm.provision "shell" do |s|		
+            s.inline = "apt-get install python-pip -y"		
+            s.privileged = true
+         end
+          list = ENV['PYTHON_REPOS'].split(' ')
+          list && list.each do |repo|
+            config.vm.provision "shell" do |s|		
+            s.inline = "cd /home/vagrant/src/#{repo} && pip install -r requirements.txt"		
```

Comment:
how long it will take ?
if long, maybe we can move it to vagrant-packer , and update another vagrant box version(0.17) for PR gate.
or like before,  install all of them in the unit-test of ucs-service job, and mount/map the workspace into vagrant.
it can be done in the future.